dockerfiles/README.md

```diff
@@ -0,0 +1,20 @@
+General
+-------
+This DockerFile contains the necessary steps to build a Docker container that can run
```

Comment:
Nice. I think in addition to this, we can write some small thing on accessing this Dockerfile in this area (so people can use it if they don't want to install it manually): https://chipyard.readthedocs.io/en/latest/Chipyard-Basics/Initial-Repo-Setup.html#initial-repository-setup. Basically something like: "Another option to use Chipyard is to use one of Chipyard's pre-build Docker images. It can be located here <LINK>. Where the image tag is the major version of Chipyard".